<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KyVanBan</class>
 <widget class="QDialog" name="KyVanBan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ký văn bản</string>
  </property>
  <widget class="QLabel" name="label_vanBanCanKy">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Văn bản cần ký</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hamBam">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Hàm băm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chuKy">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Chữ ký</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_hamBam">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_chuKy">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_taiVanBanTXT">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Tải văn bản txt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_taiVanBanDOCX">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Tải văn bản docx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_luuChuKy">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Lưu chữ ký</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_thoat">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>470</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textBrowser_vanBanCanKy">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>631</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
